src/cli/setup_mode.rs

```diff
@@ -22,7 +22,7 @@ pub fn main() -> Result<utils::ExitCode> {
         return Ok(utils::ExitCode(0));
     }
 
-    // XXX: If you change anything here, please make the same changes in rustup-init.sh
```

Comment:
The XXX's were to grab developers attentions, not a 'thing to fix' or 'TODO'; I'm not sure we need them, but I'm not sure we don't.